### ISSUE TITLE
Use a query string to set/override thebe url for dev

### ISSUE THREAD

**Issue by zischwartz** [oreillymedia/thebe#71]
In development, the url that thebe connects to changes often. I've been hackily just setting this in the html, which is an awful solution.

A good way to do this would be to set an environmental variable, but getting that into the javascript would require big changes to how the project is built.

For now, I think a better solution would to use a query string to override the url option passed when thebe is instantiated, and the cookie that gets set when it connects. 

So basically, you could do something like `http://localhost:8080/?url=192.168.99.100:8888` and thebe would try to connect to [http://192.168.99.100:8888](http://192.168.99.100:8888).
